src/fmu/settings/_resources/changelog_manager.py

```diff
@@ -25,3 +31,51 @@ def __init__(self: Self, fmu_dir: FMUDirectoryBase) -> None:
     def relative_path(self: Self) -> Path:
         """Returns the relative path to the log file."""
         return Path("logs") / LogFileName.changelog
+
+    def log_updates_to_changelog(
+        self: Self,
+        updates: dict[str, Any],
+        old_resource_dict: dict[str, Any],
+        file: str,
```

Comment:
If this takes `relative_path` from a Pydantic resource, it would have to be changed to path